data/json/items/gun/762.json

```diff
@@ -30,6 +30,38 @@
     "flags": ["MODE_BURST", "NEVER_JAMS"]
   },
   {
+    "id": "arx160",
+    "type": "GUN",
+    "symbol": "(",
```

Comment:
You need to add `arx160` to `itemgroups/guns.json`, most probably under `guns_obscure` or that's likely to get missed. I've been doing this for each item as it's migrated to the `guns/` subdir